### PR TITLE
Codejam canvas

### PR DESCRIPTION
**Pull request by PinkaCourt**
1. задание - https://github.com/rolling-scopes-school/tasks/blob/master/tasks/stage-2/codejam-canvas/codejam-canvas.md
2. скриншот - https://ibb.co/R9w1ySk
   ссылка https://inspiring-hawking-08f7ac.netlify.com/
3. 2019-10-27 / 2019-10-27
4. Total: 35
   Layout (menu, tools, canvas) - 10
   Drawing color matrixes:
	 4x4 - 10
	 32x32 - no
	Show image on canvas 5
	Dynamic canvas size switcher 10

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+  ctx.clearRect(0, 0, canvas.width, canvas.height);
+
+  let y = 0;
+
+  for (let i=0; i < data.length; i++) {
+    let x = 0;
+    let size = data.length;
+    let pixelScale = 512/size;
+    for (let j=0; j < data[0].length; j++) {
+      let color = 'rgb ' + '(' + data[i][j].slice(0,3) + ')';
```
Comment (capybartender) [inline app.js]: you don't need wthitespace after `rgb`

File: app.js [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+document.getElementById('image').onclick = function() {
+  let canvas = document.getElementById('canvas');
+  let ctx = canvas.getContext('2d');
+  ctx.clearRect(0, 0, canvas.width, canvas.height);
+  let img = new Image();
+
+  // Привязываем функцию к событию onload
+  // Это указывает браузеру, что делать, когда изображение загружено
+  img.onload = function() {
+    ctx.drawImage(img, 50, 50);
```
Comment (capybartender) [inline app.js]: please resize the image

File: app.js (hunk 1 above)
Comment (PinkaCourt) [reply app.js]: remove whitwspace

File: app.js (hunk 2 above)
Comment (PinkaCourt) [reply app.js]: change coordinates and resize image

**Pull request merged**